android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/amo/AMOAddonsProvider.kt

```diff
@@ -422,9 +421,9 @@ internal fun JSONObject.toAddon(language: String? = null): Addon {
             authors = getAuthors(),
             categories = getCategories(),
             createdAt = getSafeString("created"),
-            updatedAt = getSafeString("last_updated"),
```

Comment:
> last_updated (string) – The date of the last time the add-on was updated by its developer(s).
> From the [amo doc](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/addons.html#get--api-v4-addons-addon-(int-id|string-slug|string-guid)-) 

I guess what we were looking was the last time developer updated the add-on listing, right?

Reply:
something like that yeah. it's a mess (on the AMO side) but this PR fixes it so that the date is the same as on AMO: [Bug 1857232 - Linkify author of an add-on in the details view](https://bugzilla.mozilla.org/show_bug.cgi?id=1857232)